actions/ltc-icf-dd-deduction-fiater.vbs

```diff
@@ -264,11 +279,24 @@ budget_months = 0
 
 'Fills all budget months with "x's", so that the script will go into each one in succession.
 Do
-  EMReadScreen budget_check, 1, 12, col
-  If budget_check = "/" then
-    budget_months = budget_months + 1
-    EMWriteScreen "x", 9, col + 1
-  End if
+	If new_elig_panel = FALSE Then  'Panel prior to 1/1/25
+    	EMReadScreen budget_check, 1, 12, col
+ 	End If
+
+	If new_elig_panel = TRUE THEN 'Panel on/after 1/1/25
+		EMReadScreen budget_check, 1, 11, col
+	End If 
+
+	If budget_check = "/" then
+		budget_months = budget_months + 1
+		If new_elig_panel = FALSE Then    'Panel prior to 1/1/25
+			EMWriteScreen "x", 9, col + 1
```

Comment:
The "X" throughout this section should be uppercase to be on the safe side for formatting issues.